src/frontend/js/components/SuccessIcon/index.tsx

```diff
@@ -0,0 +1,15 @@
+export const SuccessIcon = () => {
+  return (
+    <svg
+      className="SaleTunnelStepResume__success-icon"
```

Comment:
As this component is now independent, I think we should also create a dedicated stylesheet